sdcm/sct_runner.py

```diff
@@ -600,8 +602,56 @@ def _create_instance(self,
         LOGGER.info("Got public IP: %s", instance.public_ip_address)
         self.instance = instance
 
+        # If the SCT runner instance is used as a Docker backend for tests
+        if os.getenv("SCT_CLUSTER_BACKEND") == "docker":
+            LOGGER.info("Configure unused disks to use as a Docker container volume for Scylla data.")
+            LOGGER.info("Connecting to instance...")
+            remoter = self.get_remoter(host=instance.public_ip_address, connect_timeout=120)
+
+            LOGGER.info("Getting unused disks...")
+            disks = self._get_unused_disks(remoter)
+            LOGGER.info("The following unused disks are detected: %s", disks)
+            if disks:
+                self._setup_disks(remoter, disks)
```

Comment:
Missing the part we pointing docker to those mount points

Reply:
Fixed this.
The docker now is on this additional disk:
```
ubuntu@ip-10-4-1-70:~/sct-results$ lsblk | grep mnt
nvme1n1      259:1    0 872.6G  0 disk /mnt/docker-persistent-storage

ubuntu@ip-10-4-1-70:~/sct-results$ docker inspect -f '{{ .Mounts }}' 15d4d598f324
[{bind  /var/run/docker.sock /var/run/docker.sock  rw true rprivate} {volume 93dd165165820f86c4ed24990106de312be3249acdaaaf27844ab17198c0db44 /mnt/docker-persistent-storage/volumes/93dd165165820f86c4ed24990106de312be3249acdaaaf27844ab17198c0db44/_data /var/lib/scylla local  true }]
```

Re-checked jobs:
Docker backend: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/dimakr/job/longevity-5gb-1h-docker-test/34/
AWS backend: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/dimakr/job/longevity-5gb-1h-docker-test/35/